login-workflow/docs/language-support.md

```diff
@@ -29,10 +31,11 @@ https://www.yourwebsite.com/page?lgn=fr
 ### 3. Local Storage
 
 You can also manually modify the value that is stored in Local Storage to edit or remove the stored value. Removing the value will tell the application to default to the browser default value.
+We recommend using the ChangeLanguage function or the query string rather than Local Storage to handle language modifications as any applications that are using this workflow on the same computer will have collisions.
```

Comment:
I would suggest moving this after the example and after the other callout note.
```suggestion
All applications using the @pxblue/react-auth-workflow will use the same localstorage key when doing this lookup. In order to avoid possible interference with other applications, we recommend using the ChangeLanguage function (#1) instead of relying on localstorage.
```